boards/silabs/radio_boards/slwrb4180b/Kconfig.defconfig

```diff
@@ -0,0 +1,29 @@
+# EFR32 radio board
+
+# Copyright (c) 2024 Capgemini
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_SLWRB4180B
+
+config CMU_HFXO_FREQ
+	default 38400000
+
+config CMU_LFXO_FREQ
+	default 32768
+
+config FLASH_BASE_ADDRESS
+	hex
+	default 0x0
```

Comment:
Since te flash is properly declared in `efr32mg21a020f1024im32.dtsi`, this option is not useless (see 503721bbcc6).